### DISCUSSION TITLE
Newbie help

### DISCUSSION THREAD

**Discussion by JacquesvanWyk** [LazyVim/LazyVim#2286] [General]
Hey people 
I have now started with neovim and got Lazyvim and I understand how to make plugins but say something like Neotree that is already installed how can i customize that? Say for Instance I want the neotree to right now i saw there is command to move it but If I want this as default setting where would i out this? Can I use any of the config files ? Or do i create neotree plugin myself and add settings to that?

I kind of just want to understand also what is the files for in config?

Thanks in advance

**Comment by dpetka2001** [comment]
You can just create a file in your `~/.config/nvim/lua/plugins/` directory that will contain the changes that you want to make. For example
```lua
-- ~/.config/nvim/lua/plugins/neo-tree.lua
return {
  {
    "nvim-neo-tree/neo-tree.nvim",
    opts = {
      window = {
        position = "right",
      },
    },
  },
}
```
You can see more of neo-tree's default options [here](https://github.com/nvim-neo-tree/neo-tree.nvim#longer-example-for-packer). 

The files in `/lua/config` directory should be pretty self-explanatory. `options.lua` is for most Neovim options, `keymaps.lua` for your custom keymaps that are not plugin dependant or used for lazy-loading a plugin and `autocmds.lua` for your own auto-commands.

**Reply by JacquesvanWyk** [reply]
Ok so that will add onto existing settings and override what i put there? Thanks for help

**Reply by dpetka2001** [reply]
Yep everything you put in `/lua/plugins` directory will get merged with the parent plugin spec of Lazyvim. So, your `/lua/plugins/neo-tree.lua` spec will get merged with the default neo-tree from Lazyvim. You can either add new settings of a plugin or change the ones that are default in Lazyvim.

**Reply by JacquesvanWyk** [reply]
Thanks for your time this helped me understand Lazyvim  better.
